packages/wikitext/wikitext.1.0.0/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+maintainer: "Julien SAGOT <julien.sagot@geneanet.org>"
+authors: "Maxence ABELA <maxence.abela+wikitext@gmail.com>"
+homepage: "https://github.com/geneanet/ocaml-wikitext"
+bug-reports: "https://github.com/geneanet/ocaml-wikitext/issues"
+dev-repo: "git+https://github.com/geneanet/ocaml-wikitext.git"
+synopsis: "Wikitext parser"
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+depends: [
+  "dune" { >= "1.5.1" }
```

Comment:
I have also removed the build tag from dune, there are issues with forward compatibility and cannot be used https://github.com/ocaml/opam-repository/issues/14185

Please upstream also this change